app/controllers/telegram_bot_controller.rb

```diff
@@ -0,0 +1,45 @@
+class TelegramBotController < Telegram::Bot::UpdatesController
+  include Telegram::Bot::UpdatesController::MessageContext
+  def start!(*)
+    respond_with :message, text: 'Hi. Entry your /email example@mail.com and then /password example'
+  end
+
+  def email!(email)
+    respond_with :message, text: 'Success'
+    session['email'] = email
+  end
+
+  def password!(password)
+    session['password'] = password
```

Comment:
that's not quite secure to store pwd w/o encrypting it
it'd be of better option to first validate user and simply store the `user_id` in the session, 
simply validate the pwd here and store user id in the session.. rest is similar to the devise